protos/flyteidl/core/workflow.proto

```diff
@@ -129,7 +132,8 @@ message Node {
 // Metadata for the entire workflow.
 // To be used in the future.
 message WorkflowMetadata {
-    // Default Workflow Metadata for the entire workflow.
+    // Total wait time a workflow can be delayed by queueing.
```

Comment:
What happens if a node has max wait time of 5 mins while the workflow has 1 min... does that mean we will only allow it to wait for 1min?
If there are "invalid" settings (e.g. WF queue can't be smaller than any node's queue) please update the [compiler](https://github.com/lyft/flytepropeller/tree/master/pkg/compiler) to validate and error on that.

Please also update the docs here to describe how the two interact

Reply:
In this case, I plan on following the same precedence as Resources. So in your example given Workflow would take precedence. Thanks for the link I'll update the compiler to validate!

Reply:
Sorry, there seems to be some confusion:
 Users will not be defining wait-time at the node level, they only provide wait-time for the workflow. From the wf max-wait-time (or queueing-budget, I like this term more :)), we will figure out wait-time we can afford for the nodes in the WF at runtime. Lets do a quick chat tomorrow to be on the same page.

Reply:
Updated to only be specified in wf metadata. Have no preference over `max_wait_time` or `queueing_budget`